src/dodal/devices/fast_grid_scan.py

```diff
@@ -208,35 +209,38 @@ def clean_up(self):
 class FastGridScan(Device):
     x_steps: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "X_NUM_STEPS")
     y_steps: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Y_NUM_STEPS")
-    z_steps: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z_NUM_STEPS")
+    #z_steps: EpicsSignalWithRBV = Component(EpicsSignalWithRBV, "Z_NUM_STEPS")
+    z_steps: Signal = Component(Signal, kind=Kind.hinted, value=1)
```

Comment:
Think carefully about how to make this generic.

Or duplicate the whole thing as `SingleFastGridScan` and make the old one `DualFastGridScan` to reflect the fact it does two independent grids.